server/src/app.module.ts

```diff
@@ -22,7 +22,8 @@ import { SitemapModule } from './sitemap/sitemap.module'
         RateLimiterModule.register({
             for: 'Fastify',
             points: Number.MAX_SAFE_INTEGER,
-            duration: 1
+            duration: 1,
+            whiteList: ['127.0.0.1', configService.getEnv('FRONTEND_IP')]
```

Comment:
Dont think it's a good idea to hardcode localhost here since this will be used by the production application.

We can set FRONTEND_IP = 127.0.0.1 in `.env.example` and let the engineers decide which IP they want to whitelist in their locals.

We can do the same for staging and production as well.